.scrutinizer.yml

```diff
@@ -0,0 +1,12 @@
+filter:
+    excluded_paths: [tests/*]
+build:
+    tests:
+        override:
+            -
+                command: 'vendor/bin/phpunit --coverage-clover=coverage.clover'
```

Comment:
Scrutinizer already gives a success/failure status right?:
![image](https://cloud.githubusercontent.com/assets/2146611/24394060/c62cf4f4-139a-11e7-865e-9cfa733f3031.png)

I will look into the possibility of having Scrutinizer build upon multiple PHP versions.